doc/reference/configuration/configuration_reference.rst

```diff
@@ -3133,6 +3133,35 @@ log.syslog.*
     | Default: box.NULL
     | Environment variable: TT_LOG_SYSLOG_SERVER
 
+.. _configuration_reference_lua:
+
+lua
+---
+
+The ``lua`` section outlines the configuration parameters related to the ``lua`` environment within Tarantool.
```

Comment:
"Lua" is not highlighted as code when it's a proper name (not a keyword). So "``lua`` environment" is just "Lua environment". 

Same for the ``lua.memory`` option description.